### DISCUSSION TITLE
How to set the starting directory in ~/.zshrc

### DISCUSSION THREAD

**Discussion by GeoBrodas** [ohmyzsh/ohmyzsh#9838] [Help] [answered]
Hey there,
I want to set my directory as /mnt/c/Users/Georgey
How do I do this?

**Comment by rwmitchell** [comment]
Subject and body do not agree, can you rephrase the request?

PATH is an environment variable the shell uses to find executable programs and scripts.  Do you want to add what looks to be your home directory to your PATH?  If so, suggest instead you create a ~/bin directory and put your programs there.

To add a directory to your path, this usually works:
```export PATH=$HOME/bin:$PATH```
This will put your new path in front of everything else.  Remove the /bin if you insist on adding your entire home to your path.

Do you want to set the HOME environment variable to an alternate home?   The /mnt/c/ bit looks funny, but suspect you are using some microsoft product?

**Reply by GeoBrodas** [reply]
Hey there
I am sorry, actually, I'm new to Linux
The thing is I'm using WSL and zsh, so earlier when I used to use terminal I just had to `cd desktop/file` but now in the wsl I have to type in `cd /mnt/c/Users/Georgey/Desktop` to come to access my files in Desktop.
Yes I'm using Windows, so in order to access the C drive I have to `cd /mnt/c`

I thought by setting the PATH variable I'll be able to set this to the default directory every time I open
Thanks for the reply!

**Reply by phdd0** [reply]
God i had to just suffer through a week of WSL with Ubuntu Terminal on Win10 to emulate the MacOS env all the other ruby boot camp students are using. I LOVE this new M1 machine... Couldn't be happier dumping specifically win10, which i hate to its very core. I'd been on Windows since 3.11... who says you can't teach someone half through their 40s!?

**Comment by mcornella** [comment, marked as answer]
You could add a `cd` command at the bottom of your zshrc file:
```zsh
cd /mnt/c/Users/Georgey/Desktop
```

**Reply by GeoBrodas** [reply]
Yes that's working, thanks, but is there any other way
This one is taking like 1 second to run the code

**Reply by mcornella** [reply]
If you're using Windows Terminal you can specify the starting directory of a profile. That's another way.

**Reply by HomyeeKing** [reply]
but if the IDE like VSCode use zsh as default terminal, the entry won't be the ${cwd} but the `cd <path>` you specified
how to solve this

**Reply by mcornella** [reply]
Look for a starting directory setting in your other terminal application (i.e. what I said in the previous comment).
